### PR TITLE
[ Quorum ] Added network verification steps for quorum

### PR DESCRIPTION
**Pull request by srinivasansankaran**
Signed-off-by: srinivasan.sankaran <srinivasan.sankaran@accenture.com>

**Changelog**
- Add network verification steps for Quorum

 

**Reviewed by**
@suvajit-sarkar 
@jagpreetsinghsasan 

 

**Linked issue**
#884

### CODE REVIEW

File: docs/source/operations/baf_verify.md [hunk 1]
```diff
@@ -237,3 +237,56 @@ For final checking of the validity of the indy network.
   ```
 
 Upon successful connection, should display a `Pool Connected Successfully` msg.
+
+## Quorum Checks
+
+The flow chart shows the Quorum Deployment process. To verify the steps of deployment, follow the verification Table 'Q', to troubleshoot the general errors.
+
+![](./../_static/quorum_flowchart.png)
```
Comment (jagpreetsinghsasan) [inline docs/source/operations/baf_verify.md:245]: Flowchart not visible

File: docs/source/operations/baf_verify.md [hunk 2]
```diff
@@ -237,3 +237,56 @@ For final checking of the validity of the indy network.
...
+| Q1      | h           | **Ansible playbook failed**  <br> Playbook execution terminated at <br> **Role**: create/tessera or create/constellation <br> **Task**: Create value file for Tessera TM for each node <br>  **Error**: could not locate file in lookup: network.config.genesis                                                                                                                                            | Genesis file not present in the location/not added in configuration file                                                                                                           | It happens mainly when adding new organization to the exising network. There are two ways to keep the genesis file, one is to give the genesis file path of the exising network or hardcode the base64 encoded genesis in the configuration file.                                                                                                                                                                                                                                                                                 |
```
Comment (jagpreetsinghsasan) [inline docs/source/operations/baf_verify.md:260]: Can we give the genesis file path as well?
I thought we need to hardcode the genesis file content into the network.yaml

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]

File: docs/source/operations/baf_verify.md (hunk 2 above)
Comment (srinivasansankaran) [reply docs/source/operations/baf_verify.md:260]: yes. we can give the genesis file path as well.

File: docs/source/operations/baf_verify.md (hunk 1 above)
Comment (srinivasansankaran) [reply docs/source/operations/baf_verify.md:245]: Fixed it @jagpreetsinghsasan

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM

**Pull request merged**